modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -134,6 +138,11 @@ The channels you need for this procedure are:
 | -
 | res8-manager-tools-pool-x86_64
 
+| {es} 9
+| rhel8-pool-x86_64
```

Comment:
pool label is `el9-pool-x86_64` and tools label is `el9-manager-tools-pool-x86_64`

Reply:
ok, will change it.